clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactory.java

```diff
@@ -140,6 +166,7 @@ public EhcacheClientEntity retrieve(String identifier, ServerSideConfiguration c
        */
       boolean validated = false;
       try {
+        entity.setTimeouts(entityTimeouts);
```

Comment:
Like above, could we move this inside the validate method?  (and possibly change the name of that method as a result)